src/datagrid/README.md

```diff
@@ -10,15 +10,15 @@ Underneath it uses Facebook's [fixed-data-table](https://github.com/facebook/fix
 
 ### Dependencies
 
-- classnames, fixed-data-table-2, immutable, lodash, moment, react-bootstrap, react-day-picker, react-immutable-proptypes, react-intl, react-tether, redux, tether
+- classnames, fixed-data-table-2, immutable, lodash, moment, react-bootstrap, react-day-picker, react-immutable-proptypes, react-intl, react-tether, redux
 
```

Comment:
A general description of how the grid works could be useful i.e. a paragraph describing the overall use: how to set-up the grid and how to use the grid actions and the overall design 'philosophy'.

Reply:
added